### ISSUE TITLE
Support test cases

### ISSUE THREAD

**Issue by mrunalp** [mrunalp/ocitools#7]
We can support test cases by making test_runtime.sh take in bundle path and test case json (arguments to generate) as arguments.

**Comment by mrunalp** [comment]
@zenlinTechnofreak @liangchenye

**Comment by liangchenye** [comment]
Agree, we could provide several  test cases as the 'official test case' to cover different configurations.

In OCT, we make it an easier way: use a certain parameter.
For example, if we want to test  'terminal = false', we just need to add "--rootfs=rootfs --terminal=false" , and the generator will fill the left configs by using the default setting.

**Comment by liangchenye** [comment]
The testing logic is the same.

if the test case is the test config, it is:
     ./runtime bundleURL configURL 
if the test case is the certain parameters, it is:
     ./generater configParams -o configURL
     ./runtime bundleURL configURL
